app/enterprise/changelog.md

```diff
@@ -4,6 +4,223 @@ no_search: true
 no_version: true
 ---
 
+## 2.5.0.0 (beta1)
+**Release Date** 2021/07/13
+
+### Features
+
+#### Enterprise
+- Updated banner content informing users about free mode to include the benefits of and a link for updating to Konnect.
```

Comment:
I'd rephrase this a bit to make it clear it only shows up in free mode (vs "about free mode") and change "updating" to "using" since it's not really an update as folks think about it from a version perspective.

I'd suggest something like "For users running in free mode, updated the top banner to include information on and a link to Konnect."

Reply:
ok, will update